server/src/main/java/com/defold/extender/builders/CSharpBuilder.java

```diff
@@ -91,9 +91,11 @@ public void setSdkProject(File csProject) {
     }
 
     private File writeProject() throws IOException {
+        // The cross compilation (win32) doesn't like absolute unix paths, so we use the path relative to cwd
+        Path relativeOutputDir = sourceDir.toPath().relativize(outputDir.toPath());
```

Comment:
Making sure the path starts with "../../" in order for windows tools to support it.